trillian/ctfe/handlers.go

```diff
@@ -46,7 +46,7 @@ import (
 )
 
 // TODO(drysdale): remove this flag once everything has migrated to ByRange
-var getByRange = flag.Bool("by_range", false, "Use trillian.GetEntriesByRange for get-entries processing")
+var getByRange = flag.Bool("by_range", true, "Use trillian.GetEntriesByRange for get-entries processing")
```

Comment:
How about instead deleting this flag and the `false` code?

Reply:
Leaving the flag and changing the default first provides a period of backward compatibility for people who might be running this from `master`, but maybe this isn't how people run this?

Reply:
Deleting a flag would break backwards compatibility, whereas I can get behind the argument that this doesn't.

Reply:
Hmm, this means that this flag might have to be left there, but possibly as a noop, with a suitable docstring that says it has no effect?

Might as well do that in the same PR, then?

Reply:
I'm fine with a follow-up PRs for better granularity.